server/tests/host_test.cpp

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <vector>
+#include <numeric>
+#include <map>
+#include <string>
+
+#include "../host/host.cpp"
+
+#include "../../common/encryption/encrypt.h"
+#include "../../common/encryption/serialization.h"
```

Comment:
Maybe we can include the `common` directory in the CMake to make this a bit cleaner and avoid the `../../common/...`